changelog.d/1-api-changes/WPB-10324

```diff
@@ -0,0 +1 @@
+Drop legacy notification endpoints from client API at version 9
```

Comment:
did you also create the confluence page and add it there? seems like a good idea to me, especially since we might all completely forget by the time this becomes relevant.

Reply:
I haven't yet, as I am still not 100% sure what version this should be reflected in. The latest discussion in the ticket suggests this won't be done in version 9, but in a much later version and perhaps this PR can be closed, but let's wait and see how the discussion evolves.